pallets/credit/src/lib.rs

```diff
@@ -439,13 +440,8 @@ pub mod pallet {
                     Self::_update_credit_setting(setting);
                 }
 
-                CampaignIdSwitch::<T>::insert(0, 0);
```

Comment:
`CampaignIdSwitch` not used anymore? how about remove related code.

Reply:
check the code , It is used

Reply:
I only see `CampaignIdSwitch::insert`, any other place will read this storage?

Reply:
check the read function of CampaignIdSwitch : fn campaign_id_switch